### PR TITLE
prep for implementing collectors api

### PR DESCRIPTION
**Pull request by iamigo**
- validation error if try to change status from Stopped to Paused
- add /api/v1/helpers/nouns/collectors
- fix swagger

### CODE REVIEW

File: tests/db/model/collector/update.js [hunk 1]
```diff
@@ -41,11 +42,69 @@ describe('tests/db/model/collector/update.js >', () => {
...
+  });
+
+  it('Update status [Stopped --> Paused] invalid', (done) => {
+    expect(collectorDb.status).to.be.equal('Stopped');
+    collectorDb.update({ status: 'Paused' })
+    .then((obj) => {
+      done('Expecting error');
+    })
+    .catch((err) => {
+      console.log(err);
```
Comment (pallavi2209) [inline tests/db/model/collector/update.js]: Is this intentional console log? Or do you intend to check for error?

File: tests/db/model/collector/update.js [hunk 2]
```diff
@@ -41,11 +42,69 @@ describe('tests/db/model/collector/update.js >', () => {
...
+    collectorDb.update({ status: 'Running' })
+    .then((obj) => {
+      expect(obj.status).to.be.equal('Running');
+      done();
+    })
+    .catch(done);
+  });
+
+  it('Update status [Stopped --> Paused] invalid', (done) => {
+    expect(collectorDb.status).to.be.equal('Stopped');
```
Comment (annyhe) [inline tests/db/model/collector/update.js]: do we want to add a use case for invalid: pause -> stopped?

File: tests/db/model/collector/update.js (hunk 1 above)
Comment (iamigo) [inline tests/db/model/collector/update.js]: updated, thx

File: tests/db/model/collector/update.js (hunk 2 above)
Comment (annyhe) [inline tests/db/model/collector/update.js]: what about a test for paused -> resume is ok?
test: stop -> resume is invalid

Comment (iamigo) [inline tests/db/model/collector/update.js]: I thought I would but I can see a use case for wanting to just stop a paused job.

Comment (annyhe) [inline tests/db/model/collector/update.js]: so stop -> paused is valid?

**Pull request merged**